mk/xe-linux-distribution

```diff
@@ -306,6 +306,38 @@ identify_kylin()
     write_to_output "${distro}" "${major}" "${minor}" "$(head -n 1 ${kylin_release})"
 }
 
+identify_asianux()
+{
+    asianux_release="$1"
+    local distro
+    local major
+    local minor
+
+    # distro
+    # 'Asianux Server 4.5 (Final)'
+    # 'Asianux Server 4 (Hiranya SP2)'
+    # 'Asianux Server 4 (Hiranya SP4)'
+
+    if [ ! -f "${asianux_release}" ] ; then
+	return 1
+    fi
+
+    eval $(sed -rn \
+              's/^Asianux Server ([0-9]*)\.([0-9]*) .*$/distro=asianux;major=\1;minor=\2;/gp;'`
```

Comment:
Should we match '( )'?
How about `^Asianux Server ([0-9]*)\.([0-9]*) \(.*\)$` ?